packages/rest/src/router/metadata.ts

```diff
@@ -41,101 +44,87 @@ export interface ControllerSpec {
  *
  * @decorator
  */
-export function api(spec: ControllerSpec) {
+export function api(spec?: ControllerSpec) {
```

Comment:
👎 

What's the point of calling the `@api` decorator without any spec? It's the same as not calling the decorator at all, which should be the recommended way IMO.

Reply:
Good point. Fixed.